src/main/java/ua/javaPro/berezhnoy/hillelHomeWork4/ArrayValueCalculator.java

```diff
@@ -0,0 +1,27 @@
+package ua.javaPro.berezhnoy.hillelHomeWork4;
+
+public class ArrayValueCalculator {
+
+    public static int doCalc(String[][] doubleArray) throws ArraySizeException, ArrayDataException {
```

Comment:
```suggestion
    public static int doCalc(String[][] doubleArray)  {
```
і якщо обидва unchecked (тобто нащадки RuntimeException), то їх не треба усюди декларувати через throws